script/upgrade_manually.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
```

Comment:
We’ll use Gnosis to perform upgradeAndCall, so this file may no longer be needed